snippets/mapped-table.md

```diff
@@ -0,0 +1,35 @@
+### MappedTable
+
+Generate a table with dynamic rows based on passed array of data. Optionally include headers.
+
+Conditionaly check weather to include table headers or not. Use `Array.prototype.map` to map over array of data and return `<tr>` for each entry.
```

Comment:
Change lines on fullstops to make maintaining slightly easier (this helps us have cleaner diffs like we do in other projects).
Read the above suggestion about omitting arguments as well.
What `data` is should probably be explained as well - see [here](https://github.com/30-seconds/30-seconds-of-react/pull/8/files#diff-bcaa462389dee12dde86b1542c6fd40dR7) for my suggestion.